packages/react-data-grid/src/common/types.ts

```diff
@@ -10,9 +10,14 @@ export interface Column<T = unknown> {
   //FIXME: width and left should be optional
   width: number;
   left: number;
+  hidden?: boolean;
```

Comment:
Let's group the optional booleans together, wdyt?

Reply:
We probably can, let's do it a separate PR